Dockerfile

```
@@ -475,7 +475,7 @@ FROM sdk-libc as sdk-go
 
 ARG ARCH
 ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
-ARG GOVER="1.19.3"
+ARG GOVER="1.19.4"
```

```suggestion
ARG GOVER="1.19.6"
```
This should be 1.19.6 right?

---

Whoops! needed a rebase on the right branch!! 🙈